sdk/apimanagement/arm-apimanagement/CHANGELOG.md

```diff
@@ -1,323 +1,572 @@
+- Interface ApiCreateOrUpdateProperties has a new optional parameter translateRequiredQueryParametersConduct
+- Interface ApiManagementServiceBaseProperties has a new optional parameter natGatewayState
+- Interface ApiManagementServiceBaseProperties has a new optional parameter outboundPublicIPAddresses
+- Interface ApiManagementServiceResource has a new optional parameter natGatewayState
+- Interface ApiManagementServiceResource has a new optional parameter outboundPublicIPAddresses
+- Interface ApiManagementServiceUpdateParameters has a new optional parameter natGatewayState
+- Interface ApiManagementServiceUpdateParameters has a new optional parameter outboundPublicIPAddresses
+- Interface AuthenticationSettingsContract has a new optional parameter oAuth2AuthenticationSettings
+- Interface AuthenticationSettingsContract has a new optional parameter openidAuthenticationSettings
+- Interface AuthorizationServerContract has a new optional parameter useInApiDocumentation
+- Interface AuthorizationServerContract has a new optional parameter useInTestConsole
+- Interface AuthorizationServerContractProperties has a new optional parameter useInApiDocumentation
+- Interface AuthorizationServerContractProperties has a new optional parameter useInTestConsole
+- Interface AuthorizationServerUpdateContract has a new optional parameter useInApiDocumentation
+- Interface AuthorizationServerUpdateContract has a new optional parameter useInTestConsole
+- Interface AuthorizationServerUpdateContractProperties has a new optional parameter useInApiDocumentation
+- Interface AuthorizationServerUpdateContractProperties has a new optional parameter useInTestConsole
+- Interface IdentityProviderBaseParameters has a new optional parameter clientLibrary
+- Interface IdentityProviderContract has a new optional parameter clientLibrary
+- Interface IdentityProviderCreateContract has a new optional parameter clientLibrary
+- Interface IdentityProviderUpdateParameters has a new optional parameter clientLibrary
+- Interface OpenidConnectProviderContract has a new optional parameter useInApiDocumentation
+- Interface OpenidConnectProviderContract has a new optional parameter useInTestConsole
+- Interface OpenidConnectProviderUpdateContract has a new optional parameter useInApiDocumentation
+- Interface OpenidConnectProviderUpdateContract has a new optional parameter useInTestConsole
+- Added Enum KnownAuthorizationType
+- Added Enum KnownNatGatewayState
+- Added Enum KnownOAuth2GrantType
+- Added Enum KnownPolicyFragmentContentFormat
+- Added Enum KnownPortalSettingsCspMode
+- Added Enum KnownTranslateRequiredQueryParametersConduct
+- Class ApiManagementClient has a new signature
 
-### Bugs Fixed
+**Breaking Changes**
 
-### Other Changes
+- Type of parameter versioningScheme of interface ApiVersionSetContractDetails is changed from ApiVersionSetContractDetailsVersioningScheme to VersioningScheme
+- Removed Enum KnownApiVersionSetContractDetailsVersioningScheme
```

Comment:
If there is only this breaking I would prefer to revert it back by directive. could you find the change in swagger?

Reply:
related code with current api version:https://github.com/Azure/azure-rest-api-specs/blob/main/specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2021-08-01/definitions.json#L936
related code with target api version: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/definitions.json#L1035